mongo_connector/constants.py

```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+__version__ = '2.5.0.dev0'
```

Comment:
I don't think this is necessary. If a doc manager needed to compare versions it could use parse_version from pkg_resources.